### DISCUSSION TITLE
Unable to open database file

### DISCUSSION THREAD

**Discussion by DowXor** [jkwill87/mnamer#134] [Questions]
Hello.

Thanks you for this app ! :)

I use mrename after downloading with Deluge. But, when I use the execute plugin of Deluge, I have this error message in log : 

sqlite3.OperationalError: unable to open database file

Do you have an idea of ​​this problem?

**Comment by jkwill87** [comment]
Yea, so mnamer caches its requests to API providers to avoid repeatedly fetching the same data and to avoid rate limiting. For example, if you were running mnamer against a directory containing a season of a show there would be no reason to pull the series data repeatedly for each file.

Anyway, mnamer uses sqlite for this cache. If you run `mnamer --verbose` it will tell you exactly where this file lives. Ensure that you, or whatever user the delude process runs as, has access to write to this file. If for whatever reason its not possible to write to this location you can also instruct mnamer to not use caching with the `--no-cache` flag.

Best of luck 😃

**Reply by DowXor** [reply]
Perfect ! It works !
Thank you for your reply
And Thanks you for your app ! :)
